src/rely/Rely.re

```diff
@@ -20,6 +20,7 @@ module MatcherUtils = MatcherUtils;
 module Mock = Mock;
 module Reporter = Reporter;
 module Time = Time;
+exception InvalidWhileRunning(string);
```

Comment:
Do we have an Excepctions module where we can collect all of our custom exceptions?

Reply:
I think there is something like that in CommonErrors, but I don't want that to live there. I could see making one scoped to Rely. Also you're on PTO, don't review my code (and I will stop habitually adding you)!